### DISCUSSION TITLE
It is possible to fix NumberInput in v5?

### DISCUSSION THREAD

**Discussion by 7iomka** [mantinedev/mantine#3892] [Q&A]
Hi.
I have a wrappers around most of mantine components.
All works fine, but with NumberInput I have a problem: internal value remains the same.
I don't know why, and how to make a minimal workaround for this.
Please, see the demo: https://codesandbox.io/s/sweet-nova-1hdww2?file=/src/components/number-input/number-input.component.tsx

Try to enter any value and then erase and remove the focus - label remains in the focused state because internal value not changed. I need to fix this somehow.

Update to version 6 is still only in the plans because due to the change of pixels to rems need to do a huge layer of refactoring, so update is postponed for now.

**Comment by rtivital** [comment]
As far as I know, it is not fixable in 5.x, a breaking change was made in 6.0 to address this issue.
